src/webextension/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+
+  "description": "WebCompat Go Faster Web Extension",
+  "manifest_version": 2,
+  "name": "webcompat-extension",
+  "version": "1.0",
+
+  "permissions": [
+    "tabs",
+    // TODO: Fill required URL for override rules the same as ua_overrides.js
+    // For tablog.js to have permission to inject script to web console.
+    // Example: "*://www.google.com.tw/*"
```

Comment:
Do you think it's possible to automatically populate this list somehow? I am not too keen on having to maintain two lists of domains, especially since they don't share the same format.